targets/TARGET_STM/TARGET_STM32WB/README.md

```diff
@@ -0,0 +1,314 @@
+# STM32WB family
+
+[st.com STM32WB family page](https://www.st.com/en/microcontrollers-microprocessors/stm32wb-series.html)
+
+This ST MCU family is dual-core : based on an Arm Cortex-M4 core and an Arm Cortex-M0+ core.
+
+- M4 core is dedicated to application (mbed-os)
+- M0 core is dedicated to BLE FW (only binary, source files not provided)
+
+# Supported boards
+
+## NUCLEO_WB55RG
+
+[st.com STM32WB55RG MCU page](https://www.st.com/en/microcontrollers-microprocessors/stm32wb55rg.html)
+
+[st.com NUCLEO page](https://www.st.com/en/evaluation-tools/p-nucleo-wb55.html)
+
+- Total FLASH is 1MB, but note that it is shared by M4 and M0 cores.
+    - mbed-os application size is then limited to 768 KB
+
+- RAM: 256 KB
+    - SRAM1: 192 KB
+    - SRAM2a: 32 KB
+    - SRAM2b: 32 KB
+
+
+# BLE
+
+## MBED-OS support
+
+BLE feature is based on Cordio stack and API (above HCI) implementation.
+
+Note that the BLE controller firmware running on the cortex-M0 is the same as in STcubeFW solution
+
+- within mbed, the controller will run only the below-HCI part
+- within STcubeFW (and STM32WPAN middleware), it can run also up to GATT / GAP layers ...
+
+## mbed-trace support
+
+trace group: BLE_WB
+
+example:
+````
+[INFO][BLE_WB]: WIRELESS COPROCESSOR FW VERSION ID = 1.4.0
+[INFO][BLE_WB]: WIRELESS COPROCESSOR FW STACK TYPE = 1
+[DBG ][BLE_WB]: mbox_write type:1, len:3
+[INFO][BLE_WB]: TX>> BLE CMD
+[DBG ][BLE_WB]:   Type 0x1
+[DBG ][BLE_WB]:    Cmd 0xc03
+[DBG ][BLE_WB]:    Len 0D]
+````
+
+## BLE FW update
+
+Latest information :
+https://github.com/STMicroelectronics/STM32CubeWB/blob/master/how_to_program_wireless_stacks.txt
+
+Latest BLE FW :
+https://github.com/STMicroelectronics/STM32CubeWB/blob/master/Projects/STM32WB_Copro_Wireless_Binaries/stm32wb5x_BLE_Stack_fw.bin
```

Comment:
Should we point the users to the LL-only version of the Wireless firmware (as we're using Cordio as the BLE stack)?
https://github.com/STMicroelectronics/STM32CubeWB/blob/master/Projects/STM32WB_Copro_Wireless_Binaries/stm32wb5x_BLE_HCILayer_fw.bin

Reply:
Let's discuss it with @LMESTM 
Yes, HCILayer binary file should be sufficient for mbed-os application, but I never tested it...

Reply:
The current mbed code (cordio on top of HCI) can work with the 2 firmware version;; HCILayer or full stack.  so yes we can point to users that they can use the HCI only version if they wish to. Nevertheless they will have to change the memory map on their own to benefit from the flash savings.  I'd rather not change this memory map by default to avoid people to get first negative impression in case they try mbed with the default firmware and this doesn't work out of the box ...